src/uu/stty/src/stty.rs

```diff
@@ -245,6 +245,42 @@ fn print_terminal_size(termios: &Termios, opts: &Options) -> nix::Result<()> {
     Ok(())
 }
 
+fn control_char_to_string(mut cc: nix::libc::cc_t) -> nix::Result<String> {
+    let mut prefix = "";
+    if cc == 0 {
+        return Ok("<undef>".to_string());
+    } else if cc >= 0x80 {
+        cc &= !0x80;
+        prefix = "M-";
+    }
+
+    match cc {
+        0..=0x1f => Ok(format!("{}^{}", prefix, (b'@' + cc) as char)),
+        0x20..=0x7e => Ok(format!("{}{}", prefix, cc as char)),
+        0x7f => Ok(format!("{}^?", prefix)),
+        _ => Err(nix::errno::Errno::ERANGE),
+    }
+}
+
+fn print_control_chars(termios: &Termios, opts: &Options) -> nix::Result<()> {
+    if !opts.all {
```

Comment:
Are control chars never printed? Not even if they're changed? It's fine if this is temporary, but then there should be a TODO in the code here probably.

Reply:
Ah, that's right, indeed they should be printed if they differ from default. I'll put a TODO, as those `min` and `time` special cases are a headache

Reply:
Fair enough!